components/gitpod-protocol/src/util/logging.ts

```diff
@@ -416,8 +418,15 @@ function stringifyLogItem(logItem: any): string {
  * Jsonifies Errors properly, not as {} only.
  */
 function jsonStringifyWithErrors(value: any): string {
```

Comment:
@csweichel @easyCZ in TS it is harder to implement idea with redacting nested structures, since all payload is an array of objects.  I can consider each object as a map and create a copy of it. WDYT?

Reply:
I went through real usages and most of time there is no payload when there is a payload then it is an object mostly with primitive properties, so going to treat this object as a map in golang.